### PR TITLE
Add udev rules for picotool

### PR DESCRIPTION
**Pull request by mbrunnen**
I added udev rules for devices in boot and in application mode. Probably not exhaustive.

### CODE REVIEW

**Comment by lurch** [comment]
See also #32

**Comment by mbrunnen** [comment]
> See also #32 
Ok, that's an issue, this is a pull request. We copied both from openocd, I guess.

**Comment by coolnickname** [comment]
Why has this not been merged yet?

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Linux/Mac: use your favorite package tool. For example, on Ubuntu:
 sudo apt install build-essential pkg-config libusb-1.0-0-dev
 ```
 
+On Linux you can add udev rules in order to use picotool without sudo:
+
+```console
+sudo cp udev/99-picotool.rules /etc/rules/99-picotool.rules
```
Comment (DavidEGrayson) [inline README.md:16]: I thought `/etc/udev/rules.d` was the main place to put rules like this.  I don't see /etc/rules documented [here](https://www.freedesktop.org/software/systemd/man/udev.html).

**Pull request merged**